Webex Connect Flows/v3.3/Template/Apple Messages for Business/README.MD

```diff
@@ -0,0 +1,77 @@
+# Overview of Apple Messages for Business template flows
+
+- This document covers the purpose and configuration details for the below template flows for Apple channel
+  - Inbound flow with form
+  - Unsubscribe flow
+
+## Overview of Apple Messages for Business inbound flow with form
+
+- This workflow will be triggered for every inbound customer message from the Apple Messages app.
+- The customer will be presented with a form asking for some basic details.
+- The fields in the form can be reconfigured by the flow admin as per their requirements.
+- The fields obtained from the form are extracted by using the Evaluate node and then made available to the agent by setting global/flow variables, using the Set Variable node.
+- Forms are supported only on iPhones and iPads by Apple, Mac devices do not have support.
+  - https://register.apple.com/resources/messages/msp-rest-api/type-interactive#form-message
+- Note: The chat widget on the agent's desktop will show the customer's name as was available the first time a message was sent from that customer to the business's asset.
+  - If the customer's name is unavailable the first time the customer messages the asset, fallback name can be updated in the flows by updating the `defaultCustomerName` variable.
+  - If `defaultCustomerName` is not available or empty, a substring of the customer's userId will be used in its place.
+
+### Configuring the Apple Messages for Business flow with a form
+
+#### Step 1: Setting up the form in the Apple Messages for Business node
+
+- Choose the message type as "Form Message"
+- In the form elements field, supply the json for the form configuration
+- In the sample flow, name, gender and email address are the fields presented in the form.
```

Comment:
please change the fields like we have it in the flow

Reply:
Updated